webapp/Utils.js

```diff
@@ -22,24 +22,59 @@ const Utils = {
     return id;
   },
 
-  getMetadataField (key, field) {
+  /**
+   * Retrieves the metadata object for the passed `key`.
+   *
+   * A specific attributes can be selected by passing `field`.
+   *
+   * The algorithm will iteratively test the key for each nesting in metadata.
+   *
+   * Example key: `netParams.popParams.E.numCells`.
+   * Since 'E' is not explicitly modeled in `netpyne.metadata`
+   * this algorithm implements additional logic to skip certain levels.
+   *
+   * @param {*} key key of metadata path.
+   * @param {*} field specific object attribute.
+   * @returns metadata object or specific attribute.
+   */
+  getMetadataField (key, field = null) {
```

Comment:
technically not a utils based on the latest discussions, as it's using context/state information. What I'd suggest is to refactor the naming as a "helper"

Reply:
makes sense